cores/esp32/HWCDC.cpp

```diff
@@ -289,11 +282,29 @@ int HWCDC::availableForWrite(void)
     return a;
 }
 
+static void flushTXBuffer()
+{
+    if (!tx_ring_buf) return;
+    UBaseType_t uxItemsWaiting = 0;
+    vRingbufferGetInfo(tx_ring_buf, NULL, NULL, NULL, NULL, &uxItemsWaiting);
+
+    size_t queued_size = 0;
+    uint8_t *queued_buff = (uint8_t *)xRingbufferReceiveUpTo(tx_ring_buf, &queued_size, 0, uxItemsWaiting);
+    if (queued_size && queued_buff != NULL) {
+        vRingbufferReturnItem(tx_ring_buf, (void *)queued_buff);
+    }
+    // flush internal CDC 
```

Comment:
it is still not clear how to reset internal HW Serial CDC buffer. 
As result, the very first data written to the CDC  will be kept in the internal JTAG/CDC HW FIFO. Latter written data will be dicarted while USB is unplugged and/or a CDC connection is not stablished.
As a CDC connection is started (with some Serial Monitor application), the latest written data will show up.